packages/cli/bin/designsystemet.ts

```diff
@@ -58,8 +62,8 @@ function makeTokenCommands() {
 
       const tokens = createTokens(props);
 
-      if (write) {
-        await writeTokens({ writeDir: write, tokens, themeName: theme, colors: props.colors });
+      if (!dry) {
```

Comment:
Sikkert out of scope no: vi burde igrunn kjøre `writeTokens`, men sende inn `dry` og hoppe over `fs.writeFile` / `fs.cp` / `fs.mkdir` (erstatte med loggmelding?) dersom `dry` er true

Reply:
hmm, ja, jeg gjorde det for build. Ble litt usikker her da det er mye `mkdir`, `cp`, `writeFile` og forventninger om at gitt mapper er finnes før den fortsetter. Ble litt større omskriving plutselig. 
Tenkte viktigs her var at createTokens testes, men vi får vel ikke testet generering av `$themes` og `$metadata` da

Reply:
Kunne funket om vi lager egen wrapper funksjoner rundt de enkelt som håndterer `dry=true`

Reply:
Foreslår å lage eit eige issue på det :) Men er einig i det siste forslaget.

Reply:
Eg har fikset det 😅